Pcap++/header/PfRingDeviceList.h

```diff
@@ -46,7 +46,7 @@ namespace pcpp
 		 * Return a list of all available PF_RING devices
 		 * @return a list of all available PF_RING devices
 		 */
-		inline const std::vector<PfRingDevice*>& getPfRingDevicesList() { return m_PfRingDeviceList; }
+		const std::vector<PfRingDevice*>& getPfRingDevicesList() { return m_PfRingDeviceList; } // TODO: Should this method really return a non-const reference?

```

Comment:
Pay attention on comment

Reply:
This list is read-only and shouldn't be changed by the caller so I guess the answer to your question is "no" - we should probably make it a const reference